apps/tasks/lib/components/task_status_pill_box.dart

```diff
@@ -53,6 +54,9 @@ class TaskStatusPillBox extends StatelessWidget {
         topRight: Radius.circular(borderRadius),
       ),
     );
+
+    const double paddingHorizontal = 6;
```

Comment:
```suggestion
    const double paddingHorizontal = paddingTiny;
```

add `const double paddingTiny= 6;` to constants file
or make the paddingHorizontal a attribute of the widget